fix(app): full-width jog controls

sfoster1 opened the pull request:
The jog controls were in MeasureTip and MeasureNozzle weren't expanding to their parent properly; give them a width=100%.

## Testing

Check the MeasureTip and MeasureNozzle screens (doesn't matter if in cal check or tip length, they're invariant in this regard) and make sure the jog controls look right.

b-cooper reviewed (commented):
This is a good change, the same symptom is already partially addressed in my PR #6962 , which adds styleProps to the JogControl component.  This way each instance of the JogControls component can decide how to position itself within its parent. I prefer this to hard coding `width="100%" ` into the component definition. After #6962 merges it should be a quick rebase if you agree.

sfoster1 committed:
fix(app): full-width jog controls

The jog controls were not wide enough in MeasureTip and MeasureNozzle,
where they have only Z displayed. Use the new styled props to specify
width=100%, which fixes the issue.

ahiuchingau reviewed (approved):
LGTM!
